internal/cloud_connector/message_handlers_test.go

```diff
@@ -111,13 +114,15 @@ func TestHandleOnlineMessagesUpdateExistingConnection(t *testing.T) {
 
 	incomingMessage := buildOnlineMessage(t, expectedMessageID, expecteMessageSentTime)
 
-	err := handleOnlineMessage(mqttClient, clientID, incomingMessage, &cfg, &topicBuilder, accountResolver, connectionRegistrar, connectedClientRecorder, sourcesRecorder)
+	logger := logger.Log.WithFields(logrus.Fields{})
```

Comment:
I think we can remove this line and pass `logger.Log` into the `handleOnlineMessage()` function.